### PR TITLE
Added GeoJSON support by default

### PR DESCRIPTION
**Pull request by Peretz30**
### Does any other open PR do the same thing?
No

### What issue is this PR fixing?
None

### How did you test this PR?

Tested on Android Real Device

I think that the less dependencies the better and GeoJSON support should be by default in this package.

```
import React from 'react';
import MapView, {Geojson} from 'react-native-maps';

const alcatraz = {
  type: 'FeatureCollection',
  features: [
    {
      type: 'Feature',
      properties: {},
      geometry: {
        type: 'Point',
        coordinates: [-122.42305755615234, 37.82687023785448],
      }
    }
  ]
};

const Map = props => (
  <MapView>
    <Geojson geojson={alcatraz} />
  </MapView>
);
```

### CODE REVIEW

**Comment by rborn** [comment]
@christopherdro @alvelig any thoughts on this?

File: lib/components/Geojson.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+    return points.concat(lines).concat(polygons);
+};
+
+const flatten = (prev, curr) => prev.concat(curr);
+
+const makeOverlay = (coordinates, feature) => {
+    let overlay = {
+        feature,
+        id: feature.id ? feature.id : uuid(),
```
Comment (nabati) [inline lib/components/Geojson.js]: Does this use of uuid really make sense? It will be a new uuid on every pass. Same further down.

**Comment by Peretz30** [comment]
@nabati are we really need this `GeometryCollection` type? There are no issues about that in https://github.com/frankrowe/react-native-geojson/issues for 2 years. So, I think wasting time to implement this feature is not worth it.

File: lib/components/Geojson.js (hunk 1 above)
Comment (Peretz30) [reply lib/components/Geojson.js]: Id is used for key prop of array of React components.

Comment (alvelig) [reply lib/components/Geojson.js]: Right, `uuid` doesn't make any sense here. You could use `index` in
```
overlays.map((overlay, index) => {
  key={overlay.id || index}
}
```
Also it isn't necessary in
```
<React.Fragment key={props.geojson.id || uuid()}>
```

Comment (alvelig) [reply lib/components/Geojson.js]: uuid is a quite costly operation, it shouldn't be used everywhere

**Comment by alvelig** [comment]
Considering my notes on the `uuid` use case, LGTM

**Comment by Peretz30** [comment]
Deleted uuid generation, used index instead.

**Comment by Peretz30** [comment]
So, that about to merge?

**Comment by alvelig** [comment]
LGTM
@rborn @christopherdro 🐽

**Comment by christopherdro** [comment]
LGTM. Could we update the docs and add a basic version in the example app?

**Comment by Peretz30** [comment]
@christopherdro Added geojson description to docs and examples

**Comment by Peretz30** [comment]
@christopherdro @rborn @alvelig Guys, is there anything else I should do?

**Comment by christopherdro** [comment]
@peretz30 Could you address the lint errors show in travis?

**Comment by Peretz30** [comment]
@christopherdro done

**Comment by GuillotJessica** [comment]
@Peretz30 will you add bbox feature as it is a common thing on geojson too ? but i don't think your library handles it yet

**Comment by Peretz30** [comment]
@GuillotJessica did you mean `getMapBoundaries` ? It's method of `MapView`. `Geojson` is a component that you should to put inside `MapView` as shown in example. You can use `getMapBoundaries` as before.

**Comment by GuillotJessica** [comment]
@Peretz30  i meant to display a map regarding the bbox received in a geojson not to get the bbox displayed

**Comment by Peretz30** [comment]
@GuillotJessica I am sorry, but I don't understand what you want to do. Can you, provide more details about it?

**Comment by GuillotJessica** [comment]
i mean if i provide a geojson with boundingBox, i would like the library mapBoundaries to be set accordingly

**Comment by Peretz30** [comment]
@GuillotJessica How do you want to provide geojson with boundingBox? Can you, please, post your question with some code on [StackOverflow](https://stackoverflow.com/)

**Pull request merged**